xarray/util/print_versions.py

```diff
@@ -0,0 +1,151 @@
+'''utility functions for printing version information
+
+
+see pandas/pandas/util/_print_versions.py'''
+import os
+import platform
+import sys
+import struct
+import subprocess
+import codecs
+import locale
+import importlib
+
+
+def get_sys_info():
+    "Returns system information as a dict"
+
+    blob = []
+
+    # get full commit hash
+    commit = None
+    if os.path.isdir(".git") and os.path.isdir("xarray"):
+        try:
+            pipe = subprocess.Popen('git log --format="%H" -n 1'.split(" "),
+                                    stdout=subprocess.PIPE,
+                                    stderr=subprocess.PIPE)
+            so, serr = pipe.communicate()
+        except:
+            pass
+        else:
+            if pipe.returncode == 0:
+                commit = so
+                try:
+                    commit = so.decode('utf-8')
+                except ValueError:
+                    pass
+                commit = commit.strip().strip('"')
+
+    blob.append(('commit', commit))
+
+    try:
+        (sysname, nodename, release,
+         version, machine, processor) = platform.uname()
+        blob.extend([
+            ("python", "%d.%d.%d.%s.%s" % sys.version_info[:]),
+            ("python-bits", struct.calcsize("P") * 8),
+            ("OS", "%s" % (sysname)),
+            ("OS-release", "%s" % (release)),
+            # ("Version", "%s" % (version)),
+            ("machine", "%s" % (machine)),
+            ("processor", "%s" % (processor)),
+            ("byteorder", "%s" % sys.byteorder),
+            ("LC_ALL", "%s" % os.environ.get('LC_ALL', "None")),
+            ("LANG", "%s" % os.environ.get('LANG', "None")),
+            ("LOCALE", "%s.%s" % locale.getlocale()),
+
+        ])
+    except:
+        pass
+
+    return blob
+
+
+def show_versions(as_json=False):
+    sys_info = get_sys_info()
+
+    deps = [
+        # (MODULE_NAME, f(mod) -> mod version)
+        ("xarray", lambda mod: mod.__version__),
+        ("pandas", lambda mod: mod.__version__),
+        ("numpy", lambda mod: mod.version.version),
+        ("scipy", lambda mod: mod.version.version),
```

Comment:
Can we just use `mod.__version__` for numpy and scipy? That's the more standard way to access this information.